recipes/rmm/all/conanfile.py

```diff
@@ -0,0 +1,86 @@
+import os
+import textwrap
+from pathlib import Path
+
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import export_conandata_patches, get, copy
+from conan.tools.layout import basic_layout
+
+required_conan_version = ">=1.52.0"
+
+
+class RmmConan(ConanFile):
+    name = "rmm"
+    description = "RAPIDS Memory Manager"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/rapidsai/rmm"
+    topics = ("cuda", "memory-management", "memory-allocation", "rapids", "header-only")
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
+
+    @property
+    def _min_cppstd(self):
+        return 17
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def configure(self):
+        # Thrust needs to use CUDA as the backend for RMM.
+        # Disabled for ConanCenter CI.
+        # self.options["thrust"].backend = "cuda"
+        pass
+
+    def requirements(self):
+        self.requires("thrust/1.17.2", transitive_headers=True, transitive_libs=True)
+        self.requires("spdlog/1.11.0", transitive_headers=True, transitive_libs=True)
+        self.requires("fmt/9.1.0", transitive_headers=True, transitive_libs=True)
```

Comment:
```suggestion
        self.requires("thrust/1.17.2")
        self.requires("spdlog/1.11.0")
        self.requires("fmt/9.1.0")
```
If ``package_type`` is a header-library, then it is not necessary to define the transitive_xxx, they are automatic.